analysis_templates/cms_minimal/law.cfg

```
@@ -121,6 +123,19 @@ lfn_sources: wlcg_fs_infn_redirector, wlcg_fs_global_redirector
 ; cf.CalibrateEvents: prod2
 
 
+[resources]
+
+# default sources of remote workflows
+# keys can have the same format as described above in [versions] to pinpoint specific tasks
+# values should be comma-separated strings in the form "RESOURCE=VALUE", where RESOURCE should refer
+# to a valid task parameter (e.g. max_runtime, htcondor_memory, etc.) so that VALUE can be parsed
+# by the respective parameter instance at runtime
+# same as for [versions], the order of options is important as it defines the resolution order
+# example:
+; run3_2023__cf.CalibrateEvents__nomin*: htcondor_memory=5GB
+; run3_2023__cf.CalibrateEvents: htcondor_memory=2GB
+
```

Just for clarification, this should also be possible, right?

```suggestion
; run3_2023__cf.CalibrateEvents__nomin*: htcondor_memory=5GB
; run3_2023__cf.CalibrateEvents: htcondor_memory=2GB
; run3_2023__cf.MLTraining: htcondor_memory=4GB, htcondor_gpus=1

```

---

ah I could have just read your example in the PR description

---

Yes, that's possible.